### PR TITLE
feat: add jmespath module

### LINKED ISSUE #142: JmesPath support
I think https://github.com/jmespath-community/go-jmespath would be a useful addtion to risor, I end up using `jq` or `yq` in most of my bash scripts and this could be a nice alternative for it in risor.

If you agree I can open a PR for it.

### PR DESCRIPTION
**Pull request by luisdavim**
closes #142

### CODE REVIEW

**Comment by myzie** [comment]
Nice! 

What goes into choosing between these two?
- https://github.com/jmespath-community/go-jmespath
- https://github.com/jmespath/go-jmespath

I just want to confirm the usage of the "community" one was intentional, since it seems to be the lesser used one.

**Comment by luisdavim** [comment]
The community one seems to be more actively maintained, see also https://github.com/jmespath/go-jmespath/issues/29

File: Makefile [hunk 1]
```diff
@@ -26,6 +26,10 @@ extension:
 postgres:
 	docker run --rm --name pg -p 5432:5432 -e POSTGRES_PASSWORD=pwd -d postgres
 
+.PHONY: tidy
+tidy:
+	find . -name go.mod -execdir go mod tidy \;
```
Comment (myzie) [inline Makefile:31]: Thanks for adding this.

**Review by myzie — APPROVED** [review]
Works well.

**Pull request merged**